CognitoSyncProject/Assets/AWSUnitySDK/AWSCore/Amazon.SecurityToken/AmazonSecurityTokenServiceConfig.cs

```diff
@@ -29,7 +30,7 @@ public AmazonSecurityTokenServiceConfig()
         {
             this.AuthenticationServiceName = "sts";
             if (this.RegionEndpoint == null)
-                this.RegionEndpoint = RegionEndpoint.USEast1;
```

Comment:
STS is only available in the US-East region, so even if you are using a different region for other services, the STS region should be fixed.